docker-compose.override.yml

```diff
@@ -5,6 +5,8 @@ services:
   app:
     ports:
       - "3000:3000"
+    volumes:
```

Comment:
how does this interact with the copied set from the Dockerfile?

Reply:
It replaces everything in that directory with your local repository, just like it did before the Dockerfile changes.

Reply:
i guess i'm asking why this is in the override and if it will clobber the work of the main Dockerfile when we want to use that.  docker-compose.override.yml is used by default unless you specify a specify list of files to use.